site/documentation/content/admin-guide/command-router-config.md

```diff
@@ -0,0 +1,160 @@
+
+The `HONO_COMMANDROUTER_AMQP_INSECURE_PORT_BIND_ADDRESS` variable can be used to specify the network interface that the port should be exposed on. By default the port is bound to the *loopback device* only, i.e. the port will only be accessible from the local host. This variable might be used to e.g. expose the non-TLS secured port on a local interface only, thus providing easy access from within the local network, while still requiring encrypted communication when accessed from the outside over public network infrastructure.
+
+Setting this variable to `0.0.0.0` will let the port being bound to **all** network interfaces (be careful not to expose the port unintentionally to the outside world).
+
+### Dual Port
+ 
+In test setups and some production scenarios Hono server may be configured to open one secure **and** one insecure port at the same time.
+ 
+This is achieved by configuring both ports correctly (see above). The server will fail to start if both ports are configured to use the same port number.
+
+Since the secure port may need different visibility in the network setup compared to the secure port, it has its own binding address `HONO_COMMANDROUTER_AMQP_INSECURE_PORT_BIND_ADDRESS`. 
+This can be used to narrow the visibility of the insecure port to a local network e.g., while the secure port may be visible worldwide. 
+
+### Ephemeral Ports
+
+Both the secure as well as the insecure port numbers may be explicitly set to `0`. The Command Router component will then use arbitrary (unused) port numbers determined by the operating system during startup.
+
+## Command & Control Connection Configuration
+
+The Command Router component requires a connection to the *AMQP 1.0 Messaging Network* in order to receive
+commands from downstream applications and forward them on a specific link on which the target protocol adapter will receive them.
+
+The connection is configured according to [Hono Client Configuration]({{< relref "hono-client-configuration.md" >}})
+with `HONO_COMMAND` being used as `${PREFIX}`. The properties for configuring response caching can be ignored.
+
+## Data Grid Connection Configuration
+
+The Command Router component requires either an embedded cache or a remote
+data grid, using the Infinispan Hotrod protocol to store device information.
+
+The following table provides an overview of the configuration variables and corresponding command line options for configuring the common aspects of the service:
+
+| Environment Variable<br>Command Line Option | Mandatory | Default | Description                                                             |
+| :------------------------------------------ | :-------: | :------ | :-----------------------------------------------------------------------|
+| `HONO_COMMANDROUTER_CACHE_COMMON_CACHENAME`<br>`--hono.commandRouter.cache.common.cacheName` | no | `command-router` | The name of the cache |
```

Comment:
this change of the prefix will also need to be reflected in the configuration classes, won't it?

Reply:
ah, just saw that it is ...